spec/models/customer_spec.rb

```diff
@@ -151,6 +151,32 @@
     end
   end
   
+  describe "determine if last name does not match record" do
+    context "when email matches" do
+      before(:each) do
+        @attrs = {:first_name => 'Bob', :last_name => 'Jones',
+          :email => 'bobjones@mail.com',
+          :street => '1234 Fake St',
+          :city => 'New York', :state => 'NY', :zip => '99999'
+        }
+        @old = create(:customer,@attrs)
```

Comment:
Is @old used for anything? If not then I feel like it should just be:
```suggestion
        create(:customer,@attrs)
```